### DISCUSSION TITLE
Subject-First Commit Messages

### DISCUSSION THREAD

**Discussion by aaronjensen** [aaronjensen/software-development#4] [Articles]
https://github.com/aaronjensen/software-development/blob/master/commit-messages.md

**Comment by Diti** [comment]
I just started using this in my projects, and I love how simple it makes to follow what has been going on in the repository.

I would love to read more guidelines about that commit style.

**Reply by aaronjensen** [reply]
Great, thanks for letting me know. I think what I would recommend is looking through commits in some of the Eventide project repositories: https://github.com/eventide-project/

We got the commit message format from Scott Bellware, one of the principals on that project and they've been using it for the duration of the project, I believe. Also, pay attention as you are writing and reading commit messages to what is the most important thing to convey. That's one of the keys, is how do we make the commit messages best suited for consumption by people why simultaneously emphasizing the work product (and deemphasizing the person that made the change).
